### ISSUE TITLE
Feedback on the wrong records

### ISSUE THREAD

**Issue by sambuccid** [mozilla/voice-web#1406]
I think that would be nice to have a text area to give a feedback on the wrond record.

I'm italian and I often hear the same people make the same mistakes, like for example ending the recording soon, and reject the contributions of these users.
If these users will have a feedback, they can solve theirs mistakes.

**Comment by davidak** [comment]
I think in addition to the "No" button, we should have buttons for common problems like ending too early, too quiet or wrongly pronounced. Also a comment free text field with "reject" button.

**Comment by sambuccid** [comment]
Yes the buttons for the common mistakes would be good, the only problem with the feedback is that can be discouraging for who receives the feedback, so the feedback should be constructive and not just a critique, so maybe could be good also add some tips and "guidelines" to who write the feedback

**Comment by moz-bozden** [comment]
Closing as stale...
